nix/docs.nix

```diff
@@ -48,31 +49,31 @@ let
 in
 pkgs.recurseIntoAttrs {
   papers = pkgs.recurseIntoAttrs {
-    system-f-in-agda = import ../papers/system-f-in-agda { inherit buildLatexDoc; };
-    eutxo = import ../papers/eutxo { inherit buildLatexDoc; };
-    utxoma = import ../papers/utxoma { inherit buildLatexDoc; };
-    eutxoma = import ../papers/eutxoma { inherit buildLatexDoc; };
+    system-f-in-agda = nativeOnly (import ../papers/system-f-in-agda { inherit buildLatexDoc; });
```

Comment:
maybe just make the latex builder set the meta attributes? I think it would be fine to say that all docs only build natively by default!